### PR TITLE
Split CI build into JS, Dart stable and Dart dev

### PR DESCRIPTION
**Pull request by yjbanov**

### CODE REVIEW

**Comment by kevmoo** [comment]
Then we can just kill https://github.com/angular/angular/pull/632

**Comment by yjbanov** [comment]
Done. Now I need to make this one work :)

File: .travis.yml [hunk 1]
```diff
@@ -4,24 +4,25 @@ node_js:
...
+- ./scripts/ci/install_dart.sh ${DART_CHANNEL} ${ARCH}
 - sh -e /etc/init.d/xvfb start
 - if [[ -e SKIP_TRAVIS_TESTS ]]; then { cat SKIP_TRAVIS_TESTS ; exit 0; } fi
 before_script:
 - mkdir -p $LOGS_DIR
 script:
-- ./scripts/ci/build.sh
-- ./scripts/ci/test_unit.sh
-- ./scripts/ci/test_e2e.sh
+- ./scripts/ci/build_and_test.sh
```
Comment (tbosch) [inline .travis.yml]: Pass in the mode explicitly

Comment (yjbanov) [inline .travis.yml]: Done.

**Pull request closed without merging**